### ISSUE TITLE
Bugfix: IAs should be able to be deleted (soft delete)

### ISSUE THREAD

**Issue by warrensearle** [jac-uk/admin#2040]
It should be possible to soft delete IAs.

When an IA is selected and the 'Delete' button pressed a modal asks the user to confirm they wish to delete the IA...
![image.png](https://images.zenhubusercontent.com/5e412648a39708a606c962c9/df1f36e5-bb4a-4a7c-8bc2-9ebe94206e58)

**Expected**
When the Ok button is pressed the IA is 'soft deleted' - it's status is updated to Deleted, it is removed from the current list and moved to the 'Deleted' list

**Actual**
When the Ok button is pressed the IA is not 'soft deleted' - it's status is unchanged and it remains in the current list

**Comment by warrensearle** [comment]
I have tested this on develop and it appears to work okay, from both the IAs list and also from the application view.

@tomlovesgithub and/or @drieJAC please could you test out the functionality to create and delete IAs. See if there's anything we could do to ensure it is robust.

If all seems fine then I suggest we close this issue.

**Comment by HalcyonJAC** [comment]
I have tested on develop and it seems that the data in Firestore has been changed but the UI doesn't re-render. I noticed there was a similar issue with the exercises table when upgrading to Vue3 and we solved it by forcing the table component to reload after the data is changed. Maybe we could do the same thing on this issue.

https://images.zenhubusercontent.com/201485031/9159f4e9-5b0a-4f44-a438-54a9132b6d7b/screen_recording_2023_08_01_at_09_05_50.mov
